<?xml version="1.0" ?>
<launch>
  <arg name="robot" default="fr3" doc="choose your robot. Possible values: [panda, fr3]"/>
  <arg name="arm_id" default="$(arg robot)" />
  <include file="$(find franka_control_test)/launch/franka_control2.launch" pass_all_args="true"/>
  <rosparam command="load" file="$(find franka_control_test)/config/robot_controllers.yaml" subst_value="true" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="robot_OSC_controller"/>
</launch>
